src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,36 @@
+@import '/src/styles/blocks/variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0 24px;
+
+  &__total {
+    display: flex;
+  }
+
+  &__number-Of-Reviews {
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
```

Comment:
The class name `&__number-Of-Reviews` does not follow BEM naming convention. Class names should be written in lowercase. Use hyphens to separate words in a name. It's better to rename it to `&__number-of-reviews`.